src/openrct2/interface/Window.cpp

```diff
@@ -1629,7 +1629,7 @@ void window_resize_gui(int32_t width, int32_t height)
     if (titleWind != nullptr)
     {
         titleWind->windowPos.x = (width - titleWind->width) / 2;
-        titleWind->windowPos.y = height - 154;
+        titleWind->windowPos.y = height - 182;
```

Comment:
Why this change?

Reply:
The update notifier was added on top, making the menu larger. This change compensates for it.

(I wondered the same thing when reviewing, myself.)